WordPress/src/main/java/org/wordpress/android/ui/main/MainActionDiffCallback.kt

```diff
@@ -0,0 +1,24 @@
+package org.wordpress.android.ui.main
+
+import androidx.recyclerview.widget.DiffUtil.Callback
+
+class MainActionDiffCallback(
+    private val oldList: List<MainActionListItem>,
+    private val newList: List<MainActionListItem>
+) : Callback() {
+    override fun areItemsTheSame(oldItemPosition: Int, newItemPosition: Int): Boolean {
+        val newItem = newList[newItemPosition]
+        val oldItem = oldList[oldItemPosition]
+
+        return (oldItem.actionType == newItem.actionType)
```

Comment:
Since this is ready for another `MainActionListItem`, maybe it makes sense to compare `actionType` in here and do the `==` check in the `areContentsTheSame` method.

Reply:
yep, it is more general 👍; changed here 699301b085377b79ab2b7db58dae7d59d051fe7d